Accepted Revisions for SDL-0315 - Add RPC Conflict management

Akihiro-Miyazaki opened the pull request:
##Introduction
This is an update of SDL-0315 which was Accepted with Revision.
We modified the some information according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-789855946).
And, clerical error correction.

Akihiro-Miyazaki commented:
Hi, @jordynmackool -san,
Thank you for your cooperation as always.
We modified the proposal documentation according to [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-789855946).
However, since "//" sentence is not accepted for json file, we modifyed to "_comment":"// Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.",.
Is it correct?

Also, since there is a clerical error in Figure 1 image file on Motivation, we replaced image file.
The name of right side yellow lifeline changed from "Web App" to "HMI".

Could you please confirm them.
Thank you.

jordynmackool commented on proposals/0315-Add-RPC-Conflict-Management.md:
````diff
@@ -130,20 +130,21 @@ App priority priority table describes the priority for each app. When a conflict
 
 Below shows the Json example for the App priority table:
 ```json
 "app_priority":{
     "EMERGENCY": 0,
+    "_comment":"// Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.",
````
```suggestion
```js
“app_priority”:{
    “EMERGENCY”: 0, // Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.
```

jordynmackool commented on proposals/0315-Add-RPC-Conflict-Management.md:
```diff
@@ -190,6 +191,7 @@ Also, below shows the Json example for the `rpc_priority_table` adding in policy
     },
     "app_priority":{
         "EMERGENCY": 0,
+        "_comment":"// Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.",
```
```suggestion
    “EMERGENCY”: 0, // Does not need to be included, if it is included it will be ignored no matter its assigned priority. See above.
```

jordynmackool reviewed (changes requested):
Hi @Akihiro-Miyazaki, thanks for submitting. I have completed my review and suggested a few changes. In addition to the suggested changes, please see below:

- Revert the change you made to the Figure 1 image file ("Web Apps" to "HMI") as that was not an accepted revision change. If you would like to make this change, you could do so by entering a revision proposal for the Steering Committee to review. 
- In line 181, please change "json" to "js"

Let me know if you have any questions. Thanks!

Akihiro-Miyazaki commented:
Hi, @jordynmackool -san,
Thank you for your help as always.

> Revert the change you made to the Figure 1 image file ("Web Apps" to "HMI") as that was not an accepted revision change. If you would like to make this change, you could do so by entering a revision proposal for the Steering Committee to review.

We understood.
We will revert the Figure 1 image file and make a commit.
Thank you.

Akihiro-Miyazaki commented:
Hi, @jordynmackool -san,
Thank you for your cooperation as always.

We reverted the Figure 1 image file and made a commit it.
Also, we accepted your comment and made a commit them.
However, since the code display was broken, we modified the proposal documentation and made a commit it.
Could you please confirm them.
Thank you.
